reporter.go

```diff
@@ -12,21 +12,23 @@ import (
 	"github.com/wavefronthq/go-metrics-wavefront"
 )
 
-// Increment counter if report is true
+// incrementCounter increments the counter by the given value if report is true
 func incrementCounter(counter metrics.Counter, value int64, report bool) {
 	if report {
 		counter.Inc(value)
 	}
 }
 
-// Update gauge value if report is true
+// updateGaugeFloat64 increments the counter by the given value if report is true
 func updateGaugeFloat64(gauge metrics.GaugeFloat64, value float64, report bool) {
 	if report {
 		gauge.Update(value)
 	}
 }
 
-// Register the standard lambda metrics.
+// registerStandardLambdaMetrics creates counters and guages for the standard AWS Lambda metrics that are reported
```

Comment:
"gauges" instead of "guages" - spell error

Reply:
Updated